vuepress/docs/next/tutorials/ecr/deploy-components-without-bundle.md

```diff
@@ -1,46 +1,54 @@
 ---
 redirectFrom: /next/tutorials/ecr/tutorials/ecr-deploy-use-plugin-and-mfe-without-bundle.html
 ---
-# Use Blueprint Generated Plugin and Micro Frontends Without a Bundle
+# Apply Entando Blueprint Without a Bundle
 
 ## Overview
 
-In this tutorial you will learn how to use the plugin and microfrontend generated using the Entando Blueprint in a running cluster without the need to include the components in a bundle.
-
-**If you haven't already generated a plugin and micro frontends with the Component Generator go here first [Generate a Plugin](../backend-developers/generate-microservices-and-micro-frontends.md)**
+In this tutorial you will learn to apply an Entando Blueprint to a plugin and Micro Frontend generated with the Entando Component Generator in a running cluster, without the need for a bundle.
 
 ## Prerequisites
 
--   An Entando Plugin built with the Entando Component Generator and populated with micro frontends
+-   An Entando Plugin built with the Entando Component Generator and populated with Micro 
+Frontends. [Generate a plugin and Micro Frontend here](../backend-developers/generate-microservices-and-micro-frontends.md). 
+
+-   Node and NPM installed on your machine (use LTS version)
 
--   Node and NPM are installed on your machine (use LTS version)
+-   Docker installed on your machine and able to upload images to Docker Hub or an image repository of your choice
```

Comment:
can leave out "on your machine"
"able to upload images to an image repository (e.g. Docker Hub)"

Reply:
reworded